src/Datadog.Trace.ClrProfiler.Managed/Integrations/Testing/NUnitIntegration.cs

```diff
@@ -232,11 +229,11 @@ private static Scope CreateScope(object testExecutionContext, Type testMethodCom
                         span.SetTag(TestTags.Type, TestTags.TypeTest);
                         CIEnvironmentValues.DecorateSpan(span);
 
-                        span.SetTag(CommonTags.RuntimeName, _runtimeDescription.Name);
-                        span.SetTag(CommonTags.RuntimeOSArchitecture, _runtimeDescription.OSArchitecture);
-                        span.SetTag(CommonTags.RuntimeOSPlatform, _runtimeDescription.OSPlatform);
-                        span.SetTag(CommonTags.RuntimeProcessArchitecture, _runtimeDescription.ProcessArchitecture);
-                        span.SetTag(CommonTags.RuntimeVersion, _runtimeDescription.ProductVersion);
+                        span.SetTag(CommonTags.RuntimeName, FrameworkDescription.Instance.Name);
+                        span.SetTag(CommonTags.RuntimeOSArchitecture, FrameworkDescription.Instance.OSArchitecture);
+                        span.SetTag(CommonTags.RuntimeOSPlatform, FrameworkDescription.Instance.OSPlatform);
+                        span.SetTag(CommonTags.RuntimeProcessArchitecture, FrameworkDescription.Instance.ProcessArchitecture);
+                        span.SetTag(CommonTags.RuntimeVersion, FrameworkDescription.Instance.ProductVersion);
```

Comment:
Probably negligible, but we can save the reference to a local var and avoid calling the property getter five times (especially since we disable method inlining).

```suggestion
                        var framework = FrameworkDescription.Instance;

                        span.SetTag(CommonTags.RuntimeName, framework.Name);
                        span.SetTag(CommonTags.RuntimeOSArchitecture, framework.OSArchitecture);
                        span.SetTag(CommonTags.RuntimeOSPlatform, framework.OSPlatform);
                        span.SetTag(CommonTags.RuntimeProcessArchitecture, framework.ProcessArchitecture);
                        span.SetTag(CommonTags.RuntimeVersion, framework.ProductVersion);
```